app/js/components/LabOrdersList.jsx

```diff
@@ -14,18 +14,19 @@ import moment from 'moment';
 import { withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
+import swal from 'sweetalert';
 import { SortableTable, Loader, constantsActions } from '@openmrs/react-components';
 import LabOrderListFilters from './LabOrdersListFilters';
 import EncounterDisplay from './EncounterDisplay';
-import { fetchLabOrders } from '../actions/labOrdersAction';
+import { fetchLabOrders, updateOrder } from '../actions/labOrdersAction';
 import { setSelectedConcept } from '../actions/labConceptsAction';
 import { filterThrough, calculateTableRows } from '../utils/helpers';
 import filtersAction from '../actions/filtersAction';
 import patientAction from '../actions/patientAction';
 import "../../css/lab-orders-list.scss";
 
 
```

Comment:
Maybe Cell should be in another file and imported ?

Reply:
yeah, @paulvazbad I also agree with moving Cell to another file, but if we did that for every Tabular component we had in this project,  we would end up with a lot of `Cell` files for each List or we would have folders in the components dir having a Cell and  a List component and we wouldn't want that